tests/integration/Core/Framework/Adapter/Command/CacheClearAllCommandTest.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Integration\Core\Framework\Adapter\Command;
+
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Adapter\Cache\CacheClearer;
+use Shopware\Core\Framework\Adapter\Command\CacheClearAllCommand;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelTestBehaviour;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * @internal
+ */
+class CacheClearAllCommandTest extends TestCase
```

Comment:
Can we make this a unit test? Since it's just calling a service I don't think we need an integration test

Reply:
Done